test/TrafficInjector/MessageSerializer.h

```diff
@@ -19,8 +19,10 @@
 #ifndef _EPROSIMA_FASTDDS_STATISTICS_BACKEND_TEST_MESSAGE_SERIALIZER_HPP_
 #define _EPROSIMA_FASTDDS_STATISTICS_BACKEND_TEST_MESSAGE_SERIALIZER_HPP_
 
-#include <fastdds_statistics_backend/topic_types/types.h>
-#include <fastdds_statistics_backend/topic_types/monitorservice_types.h>
+#include <fastdds_statistics_backend/topic_types/types.hpp>
+
+#include <fastdds_statistics_backend/topic_types/monitorservice_types.hpp>
+
```

Comment:
Remove unnecessary line breaks
```suggestion
#include <fastdds_statistics_backend/topic_types/types.hpp>
#include <fastdds_statistics_backend/topic_types/monitorservice_types.hpp>
```